content/docs/command-reference/params/index.md

```diff
@@ -130,9 +126,12 @@ epochs = params['train']['epochs']
 layers = params['train']['layers']
 ```
 
-> Note that the popular PyYAML library does not support YAML 1.2. The
-> ruamel.yaml library should be used instead to avoid subtle differences in
-> number handling.
+<admon>
+
+We use [ruamel.yaml](https://pypi.org/project/ruamel.yaml/) which supports YAML
+1.2 (unlike the more popular PyYAML).
+
+</admon>
```

Comment:
The other notes I just edited a little and made into proper admonitions

<img width="856" alt="image" src="https://user-images.githubusercontent.com/1477535/171299930-84519db9-588b-4ce2-b213-4cf92c607602.png">